### DISCUSSION TITLE
Discuss: Semantic Data Export and Exploraton

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#53] [Ideas]
Discuss, or Question and answer about this project specifically.

**Comment by Apurv428** [comment]
Hi @birm , what are the preferred tech stacks for semantic web or is it open to any?

**Reply by birm** [reply]
I don't have a particular stack in mind as long as it's arguably standard/common!

**Reply by Apurv428** [reply]
Okay.
